web-console/src/views/home-view/home-view.tsx

```diff
@@ -448,6 +479,18 @@ GROUP BY 1`,
           ),
           error: state.serverCountError,
         })}
+        {this.renderCard({
+          href: '#lookups',
+          icon: IconNames.PROPERTIES,
+          title: 'Lookups',
+          loading: state.lookupsCountLoading,
+          content: (
+            <>
+              <p>{pluralIfNeeded(state.lookupsCount, 'lookup')}</p>
+            </>
+          ),
+          error: state.lookupsUninitialized ? 'Lookups Uninitialized' : state.lookupsCountError,
```

Comment:
Should the panel render as `Error: Lookups Uninitialized` or could it be a slightly more friendly `Lookups Uninitialized` (if it’s not a huge hassle)

Reply:
Or better still `Lookups uninitialized` (sane case)